hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java

```diff
@@ -60,9 +67,18 @@
   public void execute(OzoneClient client, OzoneAddress address)
       throws IOException {
 
-    BucketArgs.Builder bb = new BucketArgs.Builder()
-        .setStorageType(StorageType.DEFAULT)
-        .setVersioning(false);
+    BucketArgs.Builder bb;
+    if (bucketType.equals(BucketType.OBJECT_STORE) || bucketType
+        .equals(BucketType.FSO)) {
+      bb = new BucketArgs.Builder().setStorageType(StorageType.DEFAULT)
+          .setVersioning(false).setBucketType(bucketType);
+    } else {
+      throw new IllegalArgumentException(
```

Comment:
Hope we have test validating this exception. If not then please add it.

Reply:
We have introduced an enum to check the required bucket types, now we don't need this exception hence, removed it.